ApiCompatTask/source/commandLineResult.ts

```diff
@@ -0,0 +1,50 @@
+import { TaskResult, getInput } from "azure-pipelines-task-lib";
+
+export default class CommandLineResult {
+    private _totalIssues: number;
+    private _body: string;
+    private green: string =  "\x1b[32m";
+    private yellow: string =  "\x1b[31m";
+    private red: string =  "\x1b[33m";
+    private failOnIssue: string =  getInput('failOnIssue');
+
+    get totalIssues() {
+        return  this._totalIssues;
+    }
+
+    get body() {
+        return this._body;
+    }
+
+    constructor( result: string ) {

```

Comment:
The constructor should be `constructor(result: string)`, I.E, without the empty spaces after `(` and before `)`